api/server/router/network/filter.go

```diff
@@ -73,6 +94,28 @@ func filterNetworks(nws []types.NetworkResource, filter filters.Args) ([]types.N
 		displayNet = append(displayNet, nw)
 	}
 
+	var danglingOnly bool
+	if filter.Contains("dangling") {
+		fErr := filter.WalkValues("dangling", func(fval string) error {
+			switch fval {
+			case "true", "1":
+				danglingOnly = true
+			case "0", "false":
+			default:
+				return errdefs.InvalidParameter(errors.New(`invalid value for filter 'dangling', must be "true" (or "1"), or "false" (or "0")`))
+			}
+			return nil
+		})
+		if fErr != nil {
+			return nil, fErr
+		}
+		danglingNet, err := filterNetworkByUse(displayNet, danglingOnly)
```

Comment:
Looks like `filterNetworkByUse()` never returns an error, so we may just as well remove error handling here (and remove the `error` return from `filterNetworkByUse()`

Reply:
This is done! 
```suggestion
		danglingNet, err := filterNetworkByUse(displayNet, danglingOnly)
```